api/odk/controllers/get-json-submissions.js

```diff
@@ -7,7 +7,7 @@ var aggregate = require('../helpers/aggregate-submissions');
  */
 module.exports = function (req, res, next) {
 
-    aggregate(req.params.formName, errorCallback, aggregateCallback);
+    aggregate(req, errorCallback, aggregateCallback);
```

Comment:
What's the background on the paired callbacks? Convention would be a single callback with `err, data, data2, ...` args.